src/continuous_agg.c

```diff
@@ -405,37 +405,68 @@ ts_continuous_agg_find_by_rv(const RangeVar *rv)
 	return ts_continuous_agg_find_by_relid(relid);
 }
 
+static ObjectAddress
+get_rel_by_name(Name schema, Name name, LOCKMODE mode)
+{
+	ObjectAddress addr;
+	Oid relid = InvalidOid;
+	Oid nspid = get_namespace_oid(NameStr(*schema), true);
+	if (OidIsValid(nspid))
+	{
+		relid = get_relname_relid(NameStr(*name), nspid);
+		if (OidIsValid(relid))
+			LockRelationOid(relid, mode);
+	}
+	ObjectAddressSet(addr, RelationRelationId, relid);
+	return addr;
+}
+
+static ObjectAddress
+get_rel_by_hypertable_id(int32 hypertable_id, LOCKMODE mode)
```

Comment:
```suggestion
lock_rel_by_hypertable_id(int32 hypertable_id, LOCKMODE mode)
```
could you rename this as well?